docs/farming.md

```diff
@@ -220,14 +209,16 @@ services:
       "--telemetry-url", "wss://telemetry.polkadot.io/submit/ 1",
       "--telemetry-url", "wss://telemetry.subspace.network/submit/ 1",
       "--rpc-cors", "all",
-      "--rpc-methods", "unsafe",
-      "--ws-external",
+      "--rpc-methods", "safe",
+      "--unsafe-ws-external",
       "--validator",
 # Replace `INSERT_YOUR_ID` with your node ID (will be shown in telemetry)
       "--name", "INSERT_YOUR_ID"
     ]
 
   farmer:
+    depends_on:
+      - node
```

Comment:
Btw, just had a thought, that maybe we should use health check here? Node startup sometimes takes a lot of time, so just starting a container != to start the node

Reply:
Yes, we should, nice catch!

I didn't add it just yet though because we need a way to make the health check and for that we need cURL to check WebSocket endpoint and current container image doesn't have cURL installed, so it'll be failing all the time. I'll make sure to add it later once we ship cURL in the container (feel free to PR that BTW).

Reply:
Sure, just leave todo here :)

Reply:
This is a user-facing document, I didn't want to leave TODOs in it to prevent confusion

Reply:
Then LGTM. Will create an issue for that